### DISCUSSION TITLE
Primo post in Indomus - help per miniprogetto

### DISCUSSION THREAD

**Discussion by mikelelarato** [indomus/forum#2192] [GENERALE - Discussioni generiche]
Buongiorno e grazie per la vostra pazienza e disponibilità,
sto entrando con entusiasmo nel mondo della domotica da poco tempo. L'obiettivo è arrivare a piccoli passi a definire un progetto di domotizzazione (spero sia un termine giusto) della mia futura casa. 
Per tutta una serie di motivazioni (salute, età) , mi pongo l'obiettivo di un impianto che mi consenta di tenere tutti gli impianti sotto controllo.
Il primo passo è l'attivazione di un piccolo impianto di video sorveglianza di due localetti (ma proprio "etti" "etti" :) in cui dovrò trasferire tutto quello che è contenuto nella casa che dovrò ristrutturare.
Mi immagino un impianto di un paio di telecamere per locale (situati in posti distanti l'uno dall'altro) collegate in remoto a un sistema centrale posto nella casa dove provvisoriamente abito in attesa di avviare i lavori.
Inoltre giusto per iniziare a dare concretezza a questo progetto vorrei installare dei sensori di presenza o qualcosa che rilevi e mi mandi un alert  in caso di presenze non autorizzare nei due locali. 
Poichè tutto questo che ho descritto è un primo passo, cosa mi consigliate di acquistare ?
Nei due locali per il momento non c'è niente. Internet può esserci  ricorrendo a qualche telefonino muletto o qualche vecchio router wifi.

Grazie

Mikele

**Comment by snakuzzo** [comment]
Hai provato a cercare nelle discussion ?
Se cerchi ad esempio "impianto" troverai diverse discussioni in cui si è già parlato di queste cose.
Prova a cercare prima lì.

**Reply by mikelelarato** [reply]
Si , ci sto guardando.
Post interessanti e ancora  molto tecnici per me.... ma ce la farò :).
Posso provare ad acquistare un paio di telecamere ip wireless e collegarle a un vecchio router tipo quelli della 3.
Parto da questo.
